### ISSUE TITLE
Cork tries to update Onyx when it's already did.

### ISSUE THREAD

**Issue by hyperphysin** [buresdv/Cork#404]
### What were you trying to do?

nothing

### What was the problem?

Cork tries to update Onyx when it's already done.
![Uploading Screenshot 2024-09-17 at 2.29.51 PM.jpg…]()

### Error logs

```shell
no log
```

### App Version

1.4.5

### App Acquisition

Bought

### Final checklist

- [ ] This report is about the interface, and I included screenshots of the problem

**Comment by hyperphysin** [comment]
![Screenshot 2024-09-17 at 2 29 51 PM](https://github.com/user-attachments/assets/72acdc9e-fb16-42ef-8163-a5e5b53b075b)

**Comment by buresdv** [comment]
I don't think this is an issue with Cork per se, and more something to do with Homebrew itself. The outdated packages are fetched from Homebrew directly, and Cork only processes the raw Homebrew output.

**Comment by buresdv** [comment]
Closer for inactivity
